update img and name (#8806)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/about-mcp.md

---
title: "dbt Model Context Protocol server"
sidebar_label: "dbt MCP server"
description: "Learn about the dbt MCP server"
id: "about-mcp"
---

import McpToolsFromReadme from '/snippets/_mcp-tools-from-readme.md';

# About dbt Model Context Protocol (MCP) server

As AI becomes more deeply integrated into data workflows, dbt users need a seamless way to access and integrate dbt's structured metadata and execution context effectively. This page provides an overview of dbt's MCP server, which exposes this context, supporting use cases such as conversational access to data, agent-driven automation of dbt workflows, and AI-assisted development.

> The [dbt MCP server](https://github.com/dbt-labs/dbt-mcp) provides a standardized framework that enables users to seamlessly integrate AI applications with dbt-managed data assets regardless of the underlying data platforms. This ensures consistent, governed access to models, metrics, lineage, and freshness across various AI tools.

Reviewer: ```suggestion
The [dbt MCP server](https://github.com/dbt-labs/dbt-mcp) provides a standardized framework that lets you integrate AI applications with dbt‑managed data assets across different data platforms. This ensures consistent, governed access to models, metrics, lineage, and freshness across your AI tools.
```

Reviewer: Hiya @mirnawong1 

(non-blocker) I've amended this slightly to make this more direct and a tad more user‑focused.

Kind Regards
Natalie
